docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

```diff
@@ -151,20 +153,47 @@ ${IMAGE_ID} tmp-cert-renew \
 {{< /tab >}}
 {{< /tabpane >}}
 
-3. If you have external etcd nodes, manually replace the `api-server-etcd-client.crt` and `api-server-etcd-client.key` file in `/etc/kubernetes/pki` (or `/var/lib/kubeadm/pki` in Bottlerocket) folder with the files you saved from any etcd node.
+3. If you have external etcd nodes, manually replace the `apiserver-etcd-client.crt` and `apiserver-etcd-client.key` file in `/etc/kubernetes/pki` (or `/var/lib/kubeadm/pki` in Bottlerocket) folder with the files you saved from any etcd node.
 
 4. Restart static control plane pods.
 
    - **For Ubuntu and RHEL**: temporarily move all manifest files from `/etc/kubernetes/manifests/` and wait for 20 seconds, then move the manifests back to this file location.
 
    - **For Bottlerocket**: re-enable the static pods:
    ```
-   apiclient get | jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=false 
-   apiclient get | jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=true`
+   apiclient get | apiclient exec admin jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=false 
+   apiclient get | apiclient exec admin jq -r '.settings.kubernetes["static-pods"] | keys[]' | xargs -n 1 -I {} apiclient set settings.kubernetes.static-pods.{}.enabled=true
    ```
 
    You can verify Pods restarting by running `kubectl` from your Admin machine.
 
 5. Repeat the above steps for all control plane nodes.
 
 You can similarly use the above steps to rotate a single certificate instead of all certificates.
+
+### Kubelet
+If `kubeadm certs check-expiration` is happy, but kubectl commands against the cluster fail with `x509: certificate has expired or is not yet valid`, then it's likely that the kubelet certs did not rotate. To rotate them, SSH back into one of the control plane nodes and do the following.
+
+```
+# backup certs
+cd /var/lib/kubelet
+cp -r pki pki.bak
+rm pki/*
+
+systemctl restart kubelet
+```
+
+### Post Renewal
+Once all the certificates are valid, verify the kcp object on the affected cluster(s) is not paused. If it is paused, then this usually indicates an issue with the etcd cluster. Check the logs for pods under the `etcdadm-controller-system` namespace for any errors. 
+If the logs indicate an issue with the etcd endpoints, then you need to update `spec.clusterConfiguration.etcd.endpoints` in the cluster's `kubeadmconfig` resource: `kubectl edit kcp -n eksa-system`
+
+```
+etcd:
+   external:
+     caFile: /var/lib/kubeadm/pki/etcd/ca.crt
+      certFile: /var/lib/kubeadm/pki/server-etcd-client.crt
+      endpoints:
+      - https://xxx.xxx.xxx.xxx:2379
+      - https://xxx.xxx.xxx.xxx:2379
+      - https://xxx.xxx.xxx.xxx:2379
```

Comment:
I know this is an example, but using `<xxx.xxx.xxx.xxx>` might make it more clear, or you should specifically add "Example" right above the snippet.